File: bronze/2562/c++/main.cpp
#include <iostream>

int main()
{
    int array[9] = {0};
    int max = 0, index;
    for (int i = 0; i < 9; i++) {
        std::cin >> array[i];
        if (array[i] > max) {
            max = array[i];
            index = i;
        }
    }

    std::cout << max << '\n' << index + 1;

    return 0;
}

File: bronze/2562/python/main.py

if (__name__ == "__main__"):
    array = list()
    max, index = (0, 0)
    for i in range(9):
        array.append(int(input()))
        if (array[i] > max):
            max = array[i]
            index = i
    print(max)
    print(index +1)